GOVERNANCE.md

```diff
@@ -0,0 +1,80 @@
+
+The Working Group includes the following roles:
+
+### 1.1. Participants
+
+“Participants” are those that have made Contributions to the Working Group subject to the [Community Specification Contribution Policy 1.0](https://spdx.org/licenses/Community-Spec-1.0.html).  The [AI GF Standard Working Group](https://github.com/finos/ai-governance-framework/issues) has the specific purpose of defining and releasing subsequent updates to the Standard. In practice, that means people that attend and contribute to meetings, raise issues, pull requests (to submit patches to the Standard) and reviews.
+
+#### How do you become a Participant?
+
+Becoming an AIGF Participant is as easy as attending a meeting and/or raising issues for changes you'd like see in the Standard, commenting on issues others have raised or even asking questions (which can often result in the clarification of the Standard's documentation to help others with the same questions in future).
+
+#### Register to vote
+Participants may register to vote on changes to the AIGF Standard (see [Section 2](#2decision-making) below). To do so register to the group by sending an empty email to [ai-governance-framework+subscribe@lists.finos.org] to join the enrolled voting participants group.
+
+Upon enrollment as an [AIGF voting participant](https://github.com/orgs/finos/teams/ai-governance-framework-participants), you will be invited to join the [FINOS GitHub organization](https://github.com/orgs/finos/people) and the [aigf-participants](https://github.com/orgs/finos/teams/ai-governance-framework-participants) GitHub team.
+
+- [View the AIGF Participants GitHub Team](https://github.com/orgs/finos/teams/ai-governance-framework-participants)
+
+### 1.2. Discussion Groups
+
+The Working Group may form one or more "Discussion Groups" to organize collaboration around a particular aspect of a specification. Discussion Groups are for discussion only.  Approval of all portions of a specification is subject to the consensus-based decision-making process of the AIGF Standard Working Group.
+
+### 1.3. Maintainers & Editors
+
+“Maintainers” are responsible for organizing activities around developing, maintaining, and updating the specification(s) developed by the Working Group.  Maintainers are also responsible for determining consensus and coordinating appeals.  The Working Group has designates six Maintainer(s). The Working Group may select a new or additional Maintainer(s) upon Approval of the Working Group Participants.  
+
+- [View the AIGF Maintainers GitHub Team](https://github.com/orgs/finos/teams/ai-governance-framework-maintainers)
+
+“Editors” are responsible for ensuring that the contents of the document accurately reflect the decisions that have been made by the group, and that the specification adheres to formatting and content guidelines. The Working Group will designate one or more Editor(s). The Working Group may select a new Editor upon Approval of the Working Group Participants.
```

Comment:
The [FDC3 editors group](https://github.com/orgs/finos/teams/fdc3-editors) looks very small is this needed?

@kriswest is this an active concept? (for reference, we're modelling our governance process on FDC3)